testvectors/json/test-hardware-version-invalid.json

```diff
@@ -20,5 +20,6 @@
   "psa-nonce": "AQIDBAUGBwgJCgsMDQ4PEBESExQVFhcYGRobHB0eHyA=",
   "psa-instance-id": "AQIDBAUGBwgJCgsMDQ4PEBESExQVFhcYGRobHB0eHyAh",
   "psa-verification-service-indicator": "https://psa-verifier.org",
-  "psa-profile": "PSA_IOT_PROFILE_1"
+  "psa-profile": "PSA_IOT_PROFILE_1",
+  "psa-certification-reference": "a234567890-12345"
```

Comment:
Does this test only check for invalid hardware-version or also invalid psa-cert-reference (P2) Case ?

Reply:
both

Reply:
ok